products/knownprojects/sql/aggregate/future_units_by_cd.sql

```diff
@@ -23,7 +23,9 @@ future_units_with_cd AS (
         units_gross,
```

Comment:
these are real ripe to be built with dbt

Reply:
(but that should wait till doing kpdb entirely - much prefer only the intermediate of testing with dbt, building entirely outside of it)